tests/keyboard_test.go

```diff
@@ -12,9 +12,8 @@ import (
 )
 
 func TestKeyboardPress(t *testing.T) {
-	tb := newTestBrowser(t)
-
 	t.Run("all_keys", func(t *testing.T) {
+		tb := newTestBrowser(t)
```

Comment:
What is the reason for moving a test browser inline? Just wondering :)

Reply:
We can't share a `browser` instance in tests now, since we can only create a single `browserContext` in a `browser`, that's the reason for the changes in some of the tests.

Reply:
Ah, right. Thanks for explaining!